assets/scripts/dialogs/UpgradeDialog.jsx

```diff
@@ -1,56 +1,115 @@
 import React, { Component } from 'react'
+import axios from 'axios'
+import { injectIntl, intlShape, FormattedMessage } from 'react-intl'
+
 import StripeCheckout from 'react-stripe-checkout'
-// import { FormattedMessage } from 'react-intl'
+import PropTypes from 'prop-types'
 import Dialog from './Dialog'
-import { STRIPE_API_KEY } from '../app/config'
-// import './UpgradeDialog.scss'
+import { API_URL, STRIPE_API_KEY } from '../app/config'
+import { connect } from 'react-redux'
+import './UpgradeDialog.scss'
```

Comment:
Let's reorganize the imports. (We are not great at consistency everywhere, but we try.) Generally, external packages are imported first, then modules from within the codebase. Usually they are grouped in some fashion. Here's a rough order:

- import react
- import prop-types
- import other react packages (usually related packages are grouped)
- import other packages (usually related packages are grouped)
- import constants
- import components
- import functions from non-component modules not covered elsewhere
- import redux related modules (e.g. action creators) last
- import CSS

When importing local files, files in the local directory are usually imported first. Files that come from the same subdirectories elsewhere are grouped together.

There's no technical reason for this - at this point, there shoudn't be any more side-effects from loading modules which require any specific module to be loaded before anything else. It's really meant to have some sort of consistent order for yourself and other developers when looking at different files.

I should document this, but let me know if you have comments or suggestions on this first.